index.html

```diff
@@ -1,187 +1,212 @@
 
-      <div class="skills">
-        <p>Skills</p>
-        <ul class="list">
-          <li>Codekit</li>
-          <li>Codepen</li>
-          <li>GitHub</li>
-          <li>GitLab</li>
-          <li>Terminal</li>
+      <div class="background-card">
+        <p class="project-name">Project name goes here</p>
+
+        <ul>
+          <li class="blue-bg">HTML/CSS</li>
+          <li class="blue-bg">Ruby on Rails</li>
+          <li class="blue-bg">Javascript</li>
         </ul>
+        <div class="arrow-link">
+          <a class="link" href="#">See this Project</a>
+          <img class="arrow-icon" src="icons/Union.svg" alt="arrow" />
+        </div>
       </div>
-    </section>
-  </body>
-</html>
+    </div>
+  </section>
+
+  <section>
+    <div class="about-section">
+      <h3>About</h3>
+      <p class="about-section-tag">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+        eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+        minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+        aliquip ea eiusmod
+      </p>
+      <button class="dark-bg-cv" type="button">
+        Get my CV<img class="down-icon" src="icons/ic_down.svg" alt="arrow-down" />
+      </button>
+    </div>
+
+    <div class="languages">
+      <p>Languages</p>
+      <ul class="list">
+        <li>Javascript (ES5/ES6)</li>
+        <li>Ruby</li>
+        <li>HTML</li>
+        <li>CSS</li>
+      </ul>
+    </div>
+
+    <div class="frameworks">
+      <p>Frameworks</p>
+      <ul class="list">
+        <li>Bootstrap</li>
+        <li>Ruby on rails</li>
+        <li>RSpec</li>
+        <li>Capibara</li>
+        <li>Selenium</li>
+      </ul>
+    </div>
+
+    <div class="skills">
+      <p>Skills</p>
+      <ul class="list">
+        <li>Codekit</li>
+        <li>Codepen</li>
+        <li>GitHub</li>
+        <li>GitLab</li>
+        <li>Terminal</li>
+      </ul>
+    </div>
+  </section>
+
+  <section>
+    <div class="about-section get-started">
+      <h4>Get started</h4>
+      <p class="about-section-tag-start">
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+        eiusmod tempor incididunt ut labore et dolore magna.
+      </p>
+
+      <form action="https://formspree.io/f/xrgdbeqr" method="post">
+        <ul>
+          <li>
+            <label for="name">
+              <input type="text" name="user-name" id="name" placeholder="Your name" maxlength="30" />
+            </label>
+          </li>
+          <li>
+            <label for="email">
+              <input type="email" name="user-email" id="email" placeholder="Your email" required />
+            </label>
+          </li>
+          <li>
+            <textarea placeholder="Enter your message here..." maxlength="500"></textarea>
+          </li>
+          <li>
+            <button type="button" class="dark-bg-submit">Start collaboration</button>
```

Comment:
- [ ] According to the requirements kindly Implement the button interactions (enable, hover, etc.). because that very important to make your website more reactive 💯🚀🚀🚀